addons/mail/static/src/core/common/composer.xml

```diff
@@ -49,15 +49,15 @@
                     t-ref="input-container"
                 >
                     <div t-if="!extended" t-attf-class="{{ actionsContainerClass }}">
-                        <Dropdown position="'top-start'" menuClass="'d-flex flex-column py-0'">
+                        <Dropdown t-if="partitionedActions.other.length > 0" position="'top-start'" menuClass="'d-flex flex-column py-0'">
                             <t t-call="mail.Composer.moreActions"/>
                             <t t-set-slot="content">
                                 <t t-call="mail.Composer.dropdownActions"/>
                             </t>
                         </Dropdown>
                     </div>
                     <div class="position-relative flex-grow-1">
-                        <t t-set="inputClasses" t-value="'o-mail-Composer-inputStyle form-control border-0 rounded-3'"/>
+                        <t t-set="inputClasses" t-value="{'o-mail-Composer-inputStyle form-control border-0 rounded-3': true, 'ps-1': partitionedActions.other.length === 0}"/>
```

Comment:
`.ps-1` doesn't look good: too close to input borders:
<img width="382" alt="Screenshot 2025-01-02 at 19 07 58" src="https://github.com/user-attachments/assets/8448b746-381d-4a56-b3bc-f6366594dbaa" />

`.ps-2` is better:
<img width="381" alt="Screenshot 2025-01-02 at 19 08 07" src="https://github.com/user-attachments/assets/60d766ed-812b-4afa-a8f5-73386bc12962" />